packages/host/app/services/operator-mode-state-service.ts

```diff
@@ -413,15 +417,15 @@ export default class OperatorModeStateService extends Service {
       let newStack: Stack = new TrackedArray([]);
       for (let item of stack) {
         let { format } = item;
-        let card = await this.cardService.loadModel(this, new URL(item.id));
-        if (!card) {
-          throw new Error(`cannot load card ${item.id}`);
-        }
-        newStack.push({
-          card,
+        let cardResource = getCard(this, () => item.id);
+        let stackItem = new StackItem({
+          owner: this, // ugh, not a great owner...
```

Comment:
services bascially live forever (unlike a component). this means that we'll never unsubscribe for events for this card because the consumer never dies. its probably not super efficient.